lib/widgets/screens/dashboard/dashboard.dart

```diff
@@ -45,10 +46,19 @@ class DashboardScreen extends StatelessWidget {
                     hintText: l10n.homeSearchHint,
                   ),
                 ),
-                ProfileHeader(
-                  avatarUrl: user.avatarUrl,
-                  profileMessage: _getGreeting(l10n, user),
-                  profileCompletionPercentage: 1.00,
+                TextButton(
+                  onPressed: () {
+                    Navigator.of(context).push(
+                      MaterialPageRoute(
+                        builder: (context) => const ProfileScreen(),
+                      ),
+                    );
+                  },
+                  child: ProfileHeader(
+                    avatarUrl: user.avatarUrl,
+                    profileMessage: _getGreeting(l10n, user),
+                    profileCompletionPercentage: 30,
+                  ),
```

Comment:
Can you also set up the navigation to the Profile page from the navBar (at the bottom of the screen)? You can check out main.dart to see how the other pages are set up.